scripts/nla.json

```diff
@@ -0,0 +1,13602 @@
+            "type": "char"
+          },
+          {
+            "name": "RES_CONSTRUCTION",
+            "type": "char"
+          },
+          {
+            "name": "RES_PIPES",
+            "type": "char"
+          },
+          {
+            "name": "RES_DUMPING",
+            "type": "char"
+          },
+          {
+            "name": "RES_ROADS",
+            "type": "char"
+          },
+          {
+            "name": "RES_BRIDGES",
+            "type": "char"
+          },
+          {
+            "name": "RES_SEWAGE",
+            "type": "char"
+          },
+          {
+            "name": "RES_STRING",
+            "type": "char"
+          },
+          {
+            "name": "REC_TRAILS",
+            "type": "char"
+          },
+          {
+            "name": "REC_PARKS",
+            "type": "char"
+          },
+          {
+            "name": "REC_PRIMITIVE",
+            "type": "char"
+          },
+          {
+            "name": "REC_RESORTS",
+            "type": "char"
+          },
+          {
+            "name": "REC_MARINAS",
+            "type": "char"
+          },
+          {
+            "name": "REC_TRASH",
+            "type": "char"
+          },
+          {
+            "name": "REC_FILMS",
+            "type": "char"
+          },
+          {
+            "name": "REC_STRING",
+            "type": "char"
+          },
+          {
+            "name": "AGR_CROPLAND",
+            "type": "char"
+          },
+          {
+            "name": "AGR_PASTURE",
+            "type": "char"
+          },
+          {
+            "name": "AGR_LIVESTOCK",
+            "type": "char"
+          },
+          {
+            "name": "AGR_ORCHARDS",
+            "type": "char"
+          },
+          {
+            "name": "AGR_POULTRY",
+            "type": "char"
+          },
+          {
+            "name": "AGR_FEEDLOT",
+            "type": "char"
+          },
+          {
+            "name": "AGR_WITHDRAWAL",
+            "type": "char"
+          },
+          {
+            "name": "AGR_STRING",
+            "type": "char"
+          },
+          {
+            "name": "IND_INDUSTRRIAL",
+            "type": "char"
+          },
+          {
+            "name": "IND_MINES",
+            "type": "char"
+          },
+          {
+            "name": "IND_OIL",
+            "type": "char"
+          },
+          {
+            "name": "IND_POWER",
+            "type": "char"
+          },
+          {
+            "name": "IND_LOGGING",
+            "type": "char"
+          },
+          {
+            "name": "IND_FIRE",
+            "type": "char"
+          },
+          {
+            "name": "IND_ODORS",
+            "type": "char"
+          },
+          {
+            "name": "IND_COMMERCIAL",
+            "type": "char"
+          },
+          {
+            "name": "IND_STRING",
+            "type": "char"
+          },
+          {
+            "name": "MAN_LIMING",
```

Comment:
Hey Shreyneil! 

Looks like you have this and the same field in line 506.

Reply:
Done sir,Please review.